### ISSUE TITLE
Let it Snow: Needs alt text for the greeting

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#317]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/letitsnow/

It is an image with no alt text. Technically it is hidden by snow by default, but I guess we won't provide a way to wipe away the snow via screen reader, so probably should put in alt text matching the image text in by default.

The snowman also needs either hidden: alt="" role="none" or alt text added. As it is part of the demo it is probably not presentational.
